Circuit_Playground_Brake_Light/code.py

```diff
@@ -0,0 +1,53 @@
+import time
+import math
+from adafruit_circuitplayground import cp
+
+brightness = 0
+l10 = [0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
+l50 = [0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
+       0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
+       0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
+       0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
+       0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
+
+i = 0
+cp.pixels.fill((255, 0, 0))
+
+light_on = False
+
+while True:
+    x, y, z = cp.acceleration
+
+    # moving average n=10, not super smooth, but it substantially lowers the amount of noise
```

Comment:
The rest of the code could use more explanation of what is going on and why. If you are going to go though each section of the code in the guide,  this is ok as is (though comments with more explanation never hurt)

Reply:
My plan was to explain it in the guide, but I'll add some more explanations